pipeline/terraform/main.tf

```diff
@@ -1,8 +1,33 @@
-module "catalogue_pipeline_20201111" {
+locals {
+  stacks = {
+    "20201111" = {
```

Comment:
Is it possible/easy to `-target` a dynamic module?

Reply:
Seemed to be when I was doing it yesterday – you just get the resource by square brackets instead of name

e.g. `module.stack["20201111"]` instead of `module.stack.catalogue_20201111`